java-sdk-example/src/test/java/AdvancedCreateSandboxTest.java

```diff
@@ -0,0 +1,191 @@
+import com.signadot.ApiClient;
```

Comment:
Nit: rename this and the other files to `CreateSandboxTestWithCustomPatch` everywhere to give a clear indication of what this test covers